bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v16/BitcoindV16RpcClient.scala

```diff
@@ -0,0 +1,83 @@
+package org.bitcoins.rpc.client.v16
+
+import akka.actor.ActorSystem
+import org.bitcoins.core.crypto.ECPrivateKey
+import org.bitcoins.core.protocol.transaction.Transaction
+import org.bitcoins.core.script.crypto.HashType
+import org.bitcoins.rpc.client.common.{
+  BitcoindRpcClient,
+  BitcoindVersion,
+  RpcOpts
+}
+import org.bitcoins.rpc.config.BitcoindInstance
+import org.bitcoins.rpc.jsonmodels.SignRawTransactionResult
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import org.bitcoins.rpc.serializers.JsonWriters._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * This class is compatible with version 0.16 of Bitcoin Core.
+  */
+class BitcoindV16RpcClient(override val instance: BitcoindInstance)(
+    implicit
+    actorSystem: ActorSystem)
+    extends BitcoindRpcClient(instance)
+    with V16AccountRpc
+    with V16SendRpc {
+
+  override def version: BitcoindVersion = BitcoindVersion.V16
+
+  def signRawTransaction(
+      transaction: Transaction): Future[SignRawTransactionResult] =
+    signRawTransaction(transaction, None, None, None)
+
+  private def signRawTransaction(
```

Comment:
Is there a reason this is defined in this file and not a transaction-related one?